File: 2917-count-pairs-whose-sum-is-less-than-target/count-pairs-whose-sum-is-less-than-target.py
class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        n = len(nums)
        count = 0
        for i in range(n):
           for j in range(i+1,n):
            total = nums[i] + nums[j]
            if total < target:
                count +=1

        return count
    
            

File: 2917-count-pairs-whose-sum-is-less-than-target/count-pairs-whose-sum-is-less-than-target.java
class Solution {
    public int countPairs(List<Integer> nums, int target) {
        int n = nums.size();
        int count = 0;

        for (int i = 0; i < n; i++) {
            for (int j = i + 1; j < n; j++) {
                int sum = nums.get(i) + nums.get(j);
                if (sum < target) {
                    count++;
                }
            }
        }

        return count;
    }
}